bower.json

```diff
@@ -14,6 +14,7 @@
     "angular-route": "~1.3.0",
     "angular-sanitize": "~1.3.0",
     "angular-touch": "~1.3.0",
+    "angular-bootstrap": "~0.13.2",
```

Comment:
They different libraries. angular-bootstrap is the glue of using bootstrap in an angular way